addons/mass_mailing/models/utm.py

```diff
@@ -34,6 +34,16 @@ def _compute_mailing_mail_count(self):
 
     def _compute_statistics(self):
         """ Compute statistics of the mass mailing campaign """
+        default_vals = {
+            'received_ratio': 0,
+            'opened_ratio': 0,
+            'replied_ratio': 0,
+            'bounced_ratio': 0
+        }
+        if not self.ids:
+            for campaign in self:
+                campaign.update(default_vals)
+            return {}
```

Comment:
This method does not return anything, so we should not change its signature. Simply use "return"